docs/configuration/secrets/extract-codacy-secrets.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+usage()
+{
+    echo "Usage: $0 -n <NAMESPACE>"
+    echo "Example: '$0 -n codacy' would retrieve secrets from the logs from the 'codacy' namespace of the kubernetes cluster"
+    exit 3
+}
+
+while getopts "n:" option; do
+    case ${option} in
+        n )
+            NAMESPACE=${OPTARG}
+            ;;
+        \? )
+            usage
+            ;;
+    esac
+done
+
+[ -z "$NAMESPACE" ] && usage
+
+echo "Starting secrets extraction"
+
+# Check if kubectl or microk8s.kubectl is available
+echo "Checking if kubectl is installed..."
+KUBECTL=$(which kubectl || which microk8s.kubectl)
+if [ $? -ne 0 ]; then
+    echo "kubectl not installed"
+    echo "Please install kubectl version specified in Codacy's documentation (see here - https://codacy.github.io/chart/install/) or the version used when installing your cluster"
+    echo "To install kubectl see https://kubernetes.io/docs/tasks/tools/install-kubectl/ (or https://microk8s.io/docs/ if you are running a microk8s kubernetes cluster)"
```

Comment:
Small edits to simplify the copy a bit:

```suggestion
    echo "Please install the kubectl version specified in Codacy's documentation (https://codacy.github.io/chart/install/) or the version used when installing your cluster"
    echo "To install kubectl see https://kubernetes.io/docs/tasks/tools/install-kubectl/, or https://microk8s.io/docs/ if you are running a MicroK8s kubernetes cluster"
```